sklearn/datasets/twenty_newsgroups.py

```diff
@@ -366,12 +370,18 @@ def fetch_20newsgroups_vectorized(subset="train", remove=(), data_home=None,
         If False, raise an IOError if the data is not locally available
         instead of trying to download the data from the source site.
 
-    return_X_y : boolean, default=False.
+    return_X_y : boolean, default=False
         If True, returns ``(data.data, data.target)`` instead of a Bunch
         object.
 
         .. versionadded:: 0.20
 
+    normalize : boolean, default=True
```

Comment:
I missed it the first time. Could you change `boolean` to `bool`. You can also modify this for `return_X_y` since you are already modifying the line. We soon going to make this more consistent